cloudinit/config/cc_landscape.py

```diff
@@ -151,8 +151,17 @@ def handle(name: str, cfg: Config, cloud: Cloud, args: list) -> None:
             ]
         )
     )
-    subp.subp(["landscape-config", "--silent"] + cmd_params)
-    util.write_file(LS_DEFAULT_FILE, "RUN=1\n")
+    try:
+        subp.subp(["landscape-config", "--silent", "--is-registered"], rcs=[5])
+        subp.subp(["landscape-config", "--silent"] + cmd_params)
+        util.write_file(LS_DEFAULT_FILE, "RUN=1\n")
+    except subp.ProcessExecutionError as e:
+        if e.exit_code == 0:
```

Comment:
Per the above I think we want to actually check for exit_code 5 to indicate already registered
```suggestion
        if e.exit_code == 5:
```

Reply:
Please see my explanation above

Reply:
agreed and reject this comment. I misread the rcs=[5] behavior and expectation from landscape-config exit codes.